app/models/invenio_rdm_record_converter.rb

```diff
@@ -131,9 +131,9 @@ def invenio_metadata(gf)
       "resource_type": resource_type(gf.resource_type.shift),
       "creators": gf.creator.map{ |creator| build_creator_contributor_json(creator) },
       "title": gf.title.first,
-      "additional_titles": additional_titles(gf.title),
+      "additional_titles": additional(category: "title", array: gf.title),
```

Comment:
I did not do the slicing of the array here only because of the tests. I would have to do the slicing of the arguments in the tests too. Knowing that, I thought you may have preferred the slicing to be done in the method instead. If not, just let me know and I can add your original suggestion back.

Reply:
@fenekku ☝️